paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlRecordParser.java

```diff
@@ -174,7 +174,7 @@ private List<DataField> extractFields(Table table) {
                             typeMapping);
             dataType = dataType.copy(typeMapping.containsMode(TO_NULLABLE) || column.isOptional());
 
-            rowType.field(column.name(), dataType);
+            rowType.field(column.name(), dataType, column.comment());
```

Comment:
For users using Flink MySQL CDC 2.3.x, the column.comment() code will report a method not found exception.